packages/core/src/providers/github.ts

```diff
@@ -120,19 +127,24 @@ export interface GitHubProfile {
  * :::
  */
 export default function GitHub(
-  config: OAuthUserConfig<GitHubProfile>
+  config: GitHubUserConfig
 ): OAuthConfig<GitHubProfile> {
+  const baseUrl = config.baseUrl || "https://github.com"
+  const apiBaseUrl = config.baseUrl
+    ? `${config.baseUrl}/api/v3`
```

Comment:
How often do they update the API version? Do we want to hardcode that here?

Reply:
Never. As it is equivalent to before, the v3 endpoint (`api.github.com`).

Reply:
v3 is not v1 though, so they have already changed it twice. Is there a link to documentation about this?